hoover/site/settings/common.py

```diff
@@ -152,3 +159,9 @@
 CELERY_BROKER_URL = os.getenv('SEARCH_AMQP_URL')
 
 SNOOP_FORWARD_HEADERS = ['Content-Disposition', 'Accept-Ranges', 'Content-Range', 'Content-Length']
+
+TUS_UPLOAD_DIR = os.path.join(base_dir, 'tus_upload')
```

Comment:
This is the cache for pending uploads, right? Let's put it outside the `base_dir`, in a configurable place (by default somewhere in /tmp). Also, a link pointing to the specific config documentation for django_tus at the start of this `TUS_` config block would be nice